lib/index.js

```diff
@@ -73,21 +73,20 @@ export default {
         const fileText = TextEditor.buffer.cachedText;
 
         // Is flow enabled for current file ?
-        const firstComStart = fileText.indexOf('\/*');
-        const firstComEnd = fileText.indexOf('*\/');
-        if (firstComStart !== -1 &&
-          firstComEnd !== -1 &&
-          fileText.slice(firstComStart + 2, firstComEnd).indexOf('@flow') === -1) {
+        if (!fileText || fileText.indexOf('@flow') === -1) {
           return Promise.resolve([]);
         }
 
         // Check if .flowconfig file is present
         const flowConfig = helpers.find(filePath, '.flowconfig');
         if (!flowConfig) {
-          atom.notifications.addWarning('[Linter-Flow] Missing .flowconfig file.', {
-            detail: 'To get started with Flow, run `flow init`.',
-            dismissable: true,
-          });
+          if (!this.lastConfigError || this.lastConfigError + 5 * 60 * 1000 < Date.now()) {
```

Comment:
Well since it would only be logging when there is an issue, and the user would only see this if they open Developer Tools I would probably leave it in.

Generally the idea from what I've seen is leave the console clean unless there is something wrong, or the user is running Atom in dev mode (and thus probably wants to see as much information as possible).